recipes/frogs/meta.yaml

```diff
@@ -1,53 +1,62 @@
 {% set name = "frogs" %}
-{% set version = "2.0.1" %}
+{% set version = "3.1.0" %}
+{% set phyloseq_extended_version = "0.99" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
-  number: 3
+  number: 0
   skip: true  # [py3k]
 
+# 1) Uplaod FROGS source code
+# 2) Add phyloseq-extented lib from Mahendra Mariadassou to FROGS (lib/external-lib) :
 source:
-  url: https://github.com/oinizan/FROGS/archive/v{{ version }}.tar.gz
-  md5: da41b0950a750be2038cefa717edf9df
+  - url: https://github.com/geraldinepascal/FROGS/archive/v{{ version }}.tar.gz 
+    md5: 1a72f44e82517b6839a82cd677b98858 
+  - url: https://github.com/mahendra-mariadassou/phyloseq-extended/archive/v{{ phyloseq_extended_version }}.tar.gz
+    md5: 1ec072751503eebb791e876a60433120
+    folder: lib/external-lib
```

Comment:
Is there a way to have phyloseq-extended as a separate package?

Reply:
Not really because this is really specific to Frogs